frame/balances/src/tests.rs

```diff
@@ -168,9 +168,9 @@ macro_rules! decl_tests {
 						<Balances as Currency<_>>::transfer(&1, &2, 1, AllowDeath),
 						Error::<$test, _>::LiquidityRestrictions
 					);
-					assert_noop!(
+					assert_err!(
```

Comment:
Why did you change this from noop to err?

Reply:
Im guessing because we are emitting an event and changing the state? I think this violates a rule that errors shouldnt change state?